srcpkgs/kasts/template

```diff
@@ -1,8 +1,9 @@
 # Template file for 'kasts'
 pkgname=kasts
-version=24.02.2
+version=24.05.1
 revision=1
 build_style=cmake
+build_helper=qemu
```

Comment:
Nope, add `qt6-declarative-host-tools` into `hostmakedepends` instead.

Reply:
is this valid for elisa too ?

Reply:
valid for everything.